docs/data/material/components/autocomplete/autocomplete.md

```diff
@@ -225,11 +225,13 @@ Before you can start using the Google Maps JavaScript API and Places API, you ne
 This demo has limited quotas to make API requests. When your quota exceeds, you will see the response for "Paris".
 :::
 
-## Custom Single Value Rendering
+## Single value rendering
 
-By default, when `multiple={false}`, the selected option is displayed as plain text inside the input. The `renderValue` prop allows you to customize how the selected value is rendered. This can be useful for adding custom styles, displaying additional information, or formatting the value differently.
+By default (when `multiple={false}`), the selected option is displayed as plain text inside the input.
+The `renderValue` prop allows you to customize how the selected value is rendered.
+This can be useful for adding custom styles, displaying additional information, or formatting the value differently:
```

Comment:
I don't think it's correct to use colon (`:`) here.
```suggestion
This can be useful for adding custom styles, displaying additional information, or formatting the value differently.
```